start-idsvr.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+####################################################################################################
+# A developer deployment to run the Curity Identity Server in Docker on the local computer.
+# The deployment provides a working configuration and includes native passkeys automation.
+#
+# Please ensure that the following resources are installed before running this script:
+# - Docker Desktop
+# - The envsubst tool (`brew install gettext` on macOS)
+#
+# To use native passkeys, you can expose the local instance of the Curity Identity Server via ngrok.
+# In this case, also install the following tools:
+# - ngrok
+# - The jq tool (`brew install jq` on macOS)
+####################################################################################################
+
```

Comment:
These scripts are not perfect but we consolidate automation across mobile examples.
Also I will update READMEs and website links for all of them.